packages/x-charts/src/models/axis.ts

```diff
@@ -441,7 +441,12 @@ export type PolarAxisDefaultized<
   AxisProps extends ChartsAxisProps = ChartsRotationAxisProps | ChartsRadiusAxisProps,
 > = Omit<PolarAxisConfig<S, V, AxisProps>, 'scaleType'> &
   AxisScaleConfig[S] &
-  AxisScaleComputedConfig[S];
+  AxisScaleComputedConfig[S] & {
+    /**
+     * Indicate if the axis should be considered by a tooltip with `trigger='axis'`.
```

Comment:
I wonder if "considered" isn't very clear in this context. 

What do you think about something like this?

```suggestion
     * If true, the contents of the axis will be displayed by a tooltip with `trigger='axis'`.
```